backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/TextCheckService.java

```diff
@@ -238,6 +318,37 @@ public List<Match> addIgnoredTextChecksIndividually(
         .toList();
   }
 
+  /**
+   * See test method for covered cases {@code
+   * TextCheckServiceTest#testAddNoIndexTags_variousCases()}
```

Comment:
🐝 Thanks for linking the Test, this is very helpful! It seems to have been renamed to `testAddNoIndexTags_withMultipleCases` though